grammars/css.cson

```diff
@@ -1080,7 +1080,7 @@
                 'include': '#escapes'
               }
               {
-                'match': "(?<=['\\s])[a-zA-Z*]+(-[a-zA-Z0-9*]*+)*+(?=['\\s])"
+                'match': "(?<=['\\s])[a-zA-Z*]+(-[a-zA-Z0-9*]*+)*(?=['\\s])"
```

Comment:
It looks like you missed the inner possessive match with this one compared to L1063.